src/main/java/org/opentripplanner/transit/model/site/StopLocation.java

```diff
@@ -135,6 +148,14 @@ default StopTransferPriority getPriority() {
 
   boolean isPartOfSameStationAs(StopLocation alternativeStop);
 
+  /**
+   * Returns the child locations of this location, for example StopLocations within a GroupStop.
+   */
+  @Nullable
+  default Set<StopLocation> getChildLocations() {
+    return null;
```

Comment:
Return `Set.of()`. As a policy we return empty collections, not `null` when it does not exist. I also think this should be a List. I know set guarantee all elements are unique, but `Set`s in Java are more difficult to use - and from the callers perspective an implementation detail. One of the problems with sets is that the order is not determenistic, which lead to hard to find "rare" failing unit tests. So, it is better to avoid them as API return types (ok in logic).